riscv-platform-spec.adoc

```diff
@@ -684,16 +680,18 @@ the base spec requirements.
 [cols="3,1,1", width=95%, align="center", options="header"]
 |===
 |Protocol                              | UEFI Section | Note
-|EFI_LOAD_FILE2_PROTOCOL               | 13.2       |
-|EFI_DECOMPRESS_PROTOCOL               | 19.5       |
+|EFI_LOAD_FILE2_PROTOCOL               | 13.2         |
+|EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL       | 14           |
+|EFI_PCI_IO_PROTOCOL                   | 14.4         |
 |===
 
 ===== Hardware Discovery Mechanisms
 
 ====== ACPI
 ACPI is the required mechanism for the hardware discovery and configuration.
 Server platforms are required to adhere to the RISC-V ACPI Platform Requirements
-Specification<<spec_riscv_acpi>>.
+Specification<<spec_riscv_acpi>>. Platform firmware must implement ACPI and the
```

Comment:
I & Kumar had a discussion about "implement". https://docs.google.com/document/d/1PfvLw3f-tPTRjl6Rw614O_RaA62oKeHlTfu7g6KZ4Vs/edit?disco=AAAAPk-SXW0

If you agree with the reasoning, lets change it to "Platform firmware must support ACPI"